components/raftstore/src/store/async_io/write_router.rs

```diff
@@ -1,5 +1,6 @@
 // Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
 
+// #[PerformanceCriticalPath]
```

Comment:
I believe functions in this file are expected to block.

Reply:
Yes, they should block, but the point here is to keep us aware do not add unnecessary code here.